### PR TITLE
devtools: Display root type for root updates in "what caused this update?"

### PR DESCRIPTION
**Pull request by eps1lon**
## Summary

When profiling an initial render the very first commit said that it was caused by "Anonymous".
Before:
Screenshot from 2021-10-20 18-15-45
After:
Screenshot from 2021-10-20 18-12-51

## How did you test this change?

```sh
$ yarn build-for-devtools
$ cd packages/react-devtools-extensions
$ yarn build:chrome
# load local extension in chrome
# "reload & profile" in https://5l577.csb.app/
```

### CODE REVIEW

File: packages/react-devtools-shared/src/devtools/views/Profiler/Updaters.js [hunk 1]
```diff
@@ -26,8 +27,9 @@ export default function Updaters({commitTree, updaters}: Props) {
   const children =
     updaters.length > 0 ? (
       updaters.map<React$Node>((serializedElement: SerializedElement) => {
-        const {displayName, id, key} = serializedElement;
-        const isVisibleInTree = commitTree.nodes.has(id);
+        const {displayName, id, key, type} = serializedElement;
+        const isVisibleInTree =
+          commitTree.nodes.has(id) && type !== ElementTypeRoot;
```
[the remark this answers is not in the document]
Comment (bvaughn) [reply packages/react-devtools-shared/src/devtools/views/Profiler/Updaters.js:32]: Probably okay for now.

**Review by bvaughn — APPROVED** [review]

**Comment by bvaughn** [comment]
Thanks!

**Comment by eps1lon** [comment]
@bvaughn Does this stay open until you're ready for a new devtools release or what's the plan here? Usually you immediately merged devtools PRs if I remember correctly.

**Comment by bvaughn** [comment]
Hm IIR CI was failing and I had to restart one of them to make sure it passed or something.

**Pull request merged**